torch/ao/quantization/fx/prepare.py

```diff
@@ -372,6 +367,27 @@ def get_arg_target_dtype(
     else:
         return node_name_to_target_dtype[arg.name]["output_activation_dtype"]
 
+def get_arg_target_dtype_as_input(
```

Comment:
sorry, what does this mean conceptually?  Is `get_arg_target_dtype_as_output` getting the output dtype of arg, and `get_arg_target_dtype_as_input` getting the input dtype of arg?  Or something else?

Reply:
after reading the code I get it...how about something like
`get_target_dtype_of_arg_from_previous_node` and `get_target_dtype_of_arg_in_cur_node`?

Reply:
or maybe `get_arg_target_dtype_as_output` and `get_arg_target_dtype_as_input_to_this_node`?

Reply:
yeah I feel this is a bit hard to name as well, I will use something like `get_arg_target_dtype_as_output` and `get_arg_target_dtype_as_input_to_this_node`